bundles/CoreBundle/Resources/config/pimcore/default.yaml

```diff
@@ -112,7 +112,7 @@ pimcore:
 
     security:
         encoder_factories:
-            Pimcore\Bundle\AdminBundle\Security\User\User: pimcore_admin.security.password_encoder_factory
+            Pimcore\Security\User\User: pimcore_admin.security.password_encoder_factory
```

Comment:
The `Pimcore\Bundle\AdminBundle\Security\User\User` is just deprecated so it should be still possible to use it. we should configure encode factory for both user classes.

Reply:
@dvesh3 The `Pimcore\Bundle\AdminBundle\Security\User\User` class extends `Pimcore\Security\User\User`. So this one alone should be fine. The test is via `instance of`